indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -290,6 +300,52 @@ class BookRoomModal extends React.Component {
         );
     }
 
+    renderEventItem = (event) => {
+        const start = `${moment(event.startDt).format('L LT')}`;
+        const sameDate = moment(event.startDt).isSame(event.endDt, 'date');
+        const endTime = moment(event.endDt).format('LT');
+        const end = sameDate ? `${endTime}` : `${moment(event.endDt).format('L')} ${endTime}`;
+        return {
+            text: event.title,
+            description: `${start} - ${end}`,
+            value: event.id
+        };
+    };
+
+    renderRelatedEventsDropdown = (disabled) => {
+        const {relatedEvents} = this.props;
+        const options = relatedEvents.map(this.renderEventItem);
+
+        if (!relatedEvents.length) {
+            return;
+        }
+
+        return (
+            <Segment>
+                <h3><Icon name="chain" />{Translate.string('Event')}</h3>
+                <div styleName="events-segment-description">
+                    <PluralTranslate count={relatedEvents.length}>
+                        <Singular>
+                            You have an event taking place in this room overlapping with booking times.
```

Comment:
"overlapping with booking times" sounds very weird... maybe just "You have an event taking place in this room" without actually mentioning the overlap? we show the event times anyway, right?

Reply:
yeah, i guess it will be enough